disspy/client.py

```diff
@@ -192,13 +194,15 @@ def __init__(
         self.token: str = str(token)
 
         if flags is None:
-            self.intflags = DisFlags.default()
+            self.intflags = Flags.default()
```

Comment:
```suggestion
            self.flags = Flags.default()
```
May rename variable to `flags`?